source/patterns/@aws-solutions-constructs/aws-kinesisstreams-gluejob/lib/index.ts

```diff
@@ -0,0 +1,186 @@
+/**
+ *  Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+import { CfnDatabase, CfnJob, CfnJobProps, CfnTable } from '@aws-cdk/aws-glue';
+import { CfnPolicy, Effect, IRole, Policy, PolicyStatement } from '@aws-cdk/aws-iam';
+import { Stream, StreamProps } from '@aws-cdk/aws-kinesis';
+import { Bucket } from '@aws-cdk/aws-s3';
+import { Aws, Construct } from '@aws-cdk/core';
+import * as defaults from '@aws-solutions-constructs/core';
+
+export interface KinesisStreamGlueJobProps {
+  /**
+   * Existing instance of Kineses Data Stream. If not set, it will create an instance
+   */
+  readonly existingStreamObj?: Stream;
+  /**
+   * User provided props to override the default props for the Kinesis Stream.
+   *
+   * @default - Default props are used
+   */
+  readonly kinesisStreamProps?: StreamProps | any;
+  /**
```

Comment:
Reformat the comment to be consistent with other comments, first the brief description followed by `@default -` on a separate line

Reply:
updating code documentation